rs/plotting/src/graphs/common.rs

```diff
@@ -44,6 +44,34 @@ impl Color {
     }
 }
 
+#[derive(Clone, Debug, Copy)]
+pub struct RegionF64 {
+    pub min: f64,
+    pub max: f64,
+}
+#[derive(Clone, Debug, Copy)]
+pub struct OrdinateBounds {
+    pub largest: f64,
+    pub len: f64,
+    pub data_driven: bool,
+}
```

Comment:
Documentation of these members could be helpful.

Reply:
done